conf/messages.en

```diff
@@ -257,8 +257,10 @@ hasCommodityCode.caption = About the goods
 hasCommodityCode.checkYourAnswersLabel = Do you have a commodity code for the goods?
 hasCommodityCode.error.required = Select yes if you know the commodity code
 hasCommodityCode.details.label = Help finding your commodity code
-hasCommodityCode.paragraph.1 = A commodity code is an internationally recognised reference number. It will help us identify the specific product you are importing. <br><br>You can {0} if you don’t know it.
-hasCommodityCode.paragraph.1.link = look up the commodity code for your goods (opens in new tab)
+hasCommodityCode.paragraph.1 = A commodity code is an internationally recognised reference number. It will help us identify the specific product you are importing.
+hasCommodityCode.linkText = look up the commodity code for your goods (opens in new tab)
+hasCommodityCode.youCan = You can
+hasCommodityCode.ifYouDontKnowIt = if you don't know it
```

Comment:
this can be changed to something like:

`
hasCommodityCode.paragraph.1 = A commodity code is an internationally recognised reference number. It will help us identify the specific product you are importing.

hasCommodityCode.paragraph.2 = You can {0} if you don't know it

hasCommodityCode.linkText = look up the commodity code for your goods (opens in new tab)
`

**{0}** when used can be used to insert the link as a parameter. 


**Welsh can be something like this**

```
hasCommodityCode.paragraph.1 = Mae cod nwyddau yn gyfeirnod sy’n cael ei gydnabod yn rhyngwladol. Bydd o gymorth i ni wrth adnabod y cynnyrch penodol rydych yn ei fewnforio.

hasCommodityCode.paragraph.2 = {0}, os nad ydych yn ei wybod.

hasCommodityCode.linkText = Gallwch chwilio am y cod nwyddau ar gyfer eich nwyddau (yn agor tab newydd)
```

Reply:
MessagesSpec is checking welsh keys and english keys match. To avoid splitting up the welsh message as I have no idea what the welsh would be we can still preserve the old message but split it into a new key.